src/clr-angular/utils/i18n/common-strings.default.ts

```diff
@@ -47,6 +47,9 @@ export const commonStringsDefault: ClrCommonStrings = {
   detailExpandableAriaLabel: 'Toggle more row content',
   // Alert
   alertCloseButtonAriaLabel: 'Close alert',
+  //Dropdown
+  ariaLabelDropdownOpen: 'Open dropdown',
```

Comment:
We want to keep same naming conventions, so let's make these `dropdownOpenAriaLabel` and `dropdownCloseAriaLabel`. This naming will change across the whole PR so I'm just making this one comment for that.